docs/releasenotes/index.rst

```diff
@@ -10,14 +10,16 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
-Next Release
-============
+v1.8.0
+======
 
 * The Docker plugin is now able to use datasets created directly via Flocker so long as the metadata has a matching ``"name"`` value.
```

Comment:
Could we use "The Flocker plugin for Docker" here instead of "The Docker plugin"?
We could even link directly to the plugin page..... (using `:ref:`docker-plugin``)